airflowHPC/dags/tasks.py

```diff
@@ -154,7 +157,7 @@ def prepare_gmxapi_input(
                     args=args,
                     input_files=inputs,
                     output_files=output_files,
-                    output_dir=f"{output_dir}/step_{counter}/sim_{i}",
+                    output_dir=f"{output_dir}/sim_{i}/iteration_{counter}",
```

Comment:
I changed it because we talked about having the implementation of airflowHPC have outputs as similar to those from the ensemble_md implementation as possible, and this is the convention adopted in ensemble_md. This could potentially minimize the confusion from the users who switch between ensemble_md and airflowHPC implementation and the work of code translation (especially for data analysis). (Also, I am not sure about the long-term future goal but if we would like to consider combining both implementations, having them as similar as possible could be beneficial.)

Reply:
OK I will keep it this way for now then but I might change it later when I do more thorough refactoring than I am doing here. Here I am only concerned with clarity and minimizing some code duplication.

PS I would have just made the changes directly in your MR if you had not been merging from a fork. I don't want to seem like I am trying to steal your code or credit for your code, I just wanted to do a little cleanup so that there are not merges where the repo is not in a consistent state.

Reply:
That's fine! It totally makes sense especially since there are a lot of moving parts. I am happy to create branches from your repo instead of a forked repo for potential future contributions. I don't think I have write access to push changes though, since I was not added as a collaborator. (Also, another branch to be merged (`modify_dag`) was also branched off from my forked repo/based on `add_functions`.)